pnpm-lock.yaml

```diff
@@ -473,7 +477,7 @@ importers:
       expo-status-bar: 1.1.0
       react: 17.0.1
       react-dom: 17.0.1_react@17.0.1
-      react-native: 0.64.3_gpe6tsd6gj6gjuc7hywxyjd2qm
+      react-native: 0.64.3_react@17.0.1
```

Comment:
not aware of y'all's strat for lock files. Let me know if this needs any adjustment.